src/implementation.cpp

```diff
@@ -134,4 +134,9 @@ const implementation *detect_best_supported_implementation_on_first_use::set_bes
 SIMDJSON_DLLIMPORTEXPORT const internal::available_implementation_list available_implementations{};
 SIMDJSON_DLLIMPORTEXPORT internal::atomic_ptr<const implementation> active_implementation{&internal::detect_best_supported_implementation_on_first_use_singleton};
 
+WARN_UNUSED error_code minify_string(const char *buf, size_t len, char *dst, size_t &dst_len) noexcept {
```

Comment:
Why did you call it minify_string instead of minify? Name conflict? Or futureproofing in case people want to minify other things?

Reply:
See my answer elsewhere in the comments. I think we need to discuss this.